packages/connector-alipay-native/src/index.test.ts

```diff
@@ -2,18 +2,20 @@ import {
   ConnectorError,
   ConnectorErrorCodes,
   GetConnectorConfig,
-  ValidateConfig,
-} from '@logto/connector-types';
+  validateConfig,
+} from '@logto/connector-core';
 import nock from 'nock';
 
-import AlipayNativeConnector from '.';
+import createConnector, { getAccessToken } from '.';
 import { alipayEndpoint } from './constant';
 import { mockedAlipayNativeConfig, mockedAlipayNativeConfigWithValidPrivateKey } from './mock';
-import { AlipayNativeConfig } from './types';
+import { AlipayNativeConfig, alipayNativeConfigGuard } from './types';
 
-const getConnectorConfig = jest.fn() as GetConnectorConfig;
+const getConfig = jest.fn().mockResolvedValue(mockedAlipayNativeConfigWithValidPrivateKey);
 
-const alipayNativeMethods = new AlipayNativeConnector(getConnectorConfig);
+function validator(config: unknown): asserts config is AlipayNativeConfig {
```

Comment:
We no longer need to test `validateConfig` since it is in `connector-core` now.

Reply:
Updated.